airflow/www/templates/airflow/graph.html

```diff
@@ -128,6 +129,17 @@
         'queued': false,
         'no_status': false
     };
+    const taskTip = d3.tip()
+      .attr('class', 'tooltip')
+      .style('background', 'black')
```

Comment:
Rather than defining the stlyes in the JS can we put them in a .css file (as we already have given this a class)

Reply:
hmm, really, `tooltip` is a class from bootstrap. will take a look